### PR TITLE
Add tests for AssetInput component

### PR DESCRIPTION
**Pull request by yivlad**

### CODE REVIEW

File: packages/app/src/ui/organisms/multi-selector/MultiSelector.test.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+        maxSelectedFieldName="isMaxSelected"
+        max={max}
+      />
+    </Form>
+  )
+}
+
+describe(ControlledMultiSelectorAssetInput.name, () => {
+  test('Changes value in input', async () => {
+    const value = NormalizedUnitNumber(12345)
```
Comment (oskarvu) [inline packages/app/src/ui/organisms/multi-selector/MultiSelector.test.tsx:59]: Maybe just `const value = '12345'`?

**Review by oskarvu — APPROVED** [review]

File: packages/app/src/ui/organisms/multi-selector/MultiSelector.test.tsx (hunk 1 above)
Comment (yivlad) [reply packages/app/src/ui/organisms/multi-selector/MultiSelector.test.tsx:67]: I think it's better to be consistent and have `NormalizedUnitNumber` everywhere

File: packages/app/src/ui/organisms/multi-selector/MultiSelector.tsx [hunk 2]
```diff
@@ -127,6 +127,21 @@ export function ControlledMultiSelectorAssetInput({
             setMax={setMaxValue ?? setMaxSelectedField}
             isMaxSelected={maxSelectedFieldName && isMaxSelected}
             onChange={(e) => {
+              if (e.target.value === MAX_VALUE_PLACEHOLDER.slice(0, -1)) {
+                // backspace was pressed while MAX was selected
+                ;(setMaxValue ?? setMaxSelectedField)?.()
+                return
+              }
+
+              e.target.value = e.target.value.replace(/,/g, '.')
```
Comment (krzkaczor) [inline packages/app/src/ui/organisms/multi-selector/MultiSelector.tsx:136]: I am not sure if it's good idea to extract this logic from AssetInput. Don't you think we might want to use it directly in the future?

Comment (yivlad) [reply packages/app/src/ui/organisms/multi-selector/MultiSelector.tsx]: Good point

**Review by krzkaczor — APPROVED** [review]

**Pull request merged**